core/cloudflow-runner/src/main/scala/cloudflow/runner/RunnerConfigResolver.scala

```diff
@@ -28,20 +28,26 @@ trait RunnerConfigResolver {
   final val SecretConfigFile      = "secret.conf"
 
   def makeConfig: Try[Config] = Try {
-    val configFilePathString = Option(System.getProperty("config.file")).getOrElse(s"$ConfigMountPath/$ConfigFile")
+    val configFilePathString = Option(System.getProperty("config.file")).getOrElse(s"$ConfigSecretMountPath/$ConfigFile")
     val configPath           = Paths.get(configFilePathString)
     val secretPath           = Paths.get(s"$ConfigSecretMountPath/$SecretConfigFile")
 
+    val applicationConfig = if (Files.exists(configPath)) {
```

Comment:
Can you add a comment here that this is for backwards compat or extract a function called `backwardsCompatConfig`

Reply:
done